Classes/Command/TranslationCommandController.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace Sitegeist\LostInTranslation\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Neos\ContentRepository\Domain\Model\NodeInterface;
+use Neos\ContentRepository\Domain\Service\Context;
+use Neos\Eel\FlowQuery\FlowQuery;
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Cli\CommandController;
+use Neos\Flow\Persistence\PersistenceManagerInterface;
+use Neos\Neos\Domain\Model\Site;
+use Neos\Neos\Domain\Repository\SiteRepository;
+use Neos\Neos\Domain\Service\ContentContextFactory;
+use Sitegeist\LostInTranslation\ContentRepository\NodeTranslationService;
+
+class TranslationCommandController extends CommandController
+{
+    /**
+     * @Flow\InjectConfiguration(package="Neos.Flow")
+     * @var array
+     */
+    protected $flowSettings;
+
+    /**
+     * @Flow\InjectConfiguration(path="nodeTranslation.languageDimensionName")
+     * @var string
+     */
+    protected $languageDimensionName;
+
+    /**
+     * @Flow\InjectConfiguration(package="Neos.ContentRepository", path="contentDimensions")
+     * @var array
+     */
+    protected $contentDimensionConfiguration;
+
+    /**
+     * @Flow\Inject
+     * @var SiteRepository
+     */
+    protected $siteRepository;
+
+    /**
+     * @Flow\Inject
+     * @var NodeTranslationService
+     */
+    protected $nodeTranslationService;
+
+    /**
+     * @Flow\Inject
+     * @var ContentContextFactory
+     */
+    protected $contentContextFactory;
+
+    /**
+     * @Flow\Inject
+     * @var EntityManagerInterface
+     */
+    protected $entityManager;
+
+    /**
+     * @Flow\Inject
+     * @var PersistenceManagerInterface
+     */
+    protected $persistenceManager;
+
+    /**
+     * @Flow\Internal
+     *
+     * @param  string  $siteNodeName
+     * @param  bool  $translate
+     * @return void
+     */
+    public function syncCommand(string $siteNodeName, bool $translate = false): void
```

Comment:
Aside from beeing in its own pr, does it make sense to add a language parameter here to not affect all dimensions?

Reply:
Moved to a separate PR #15